### PR TITLE
Updated content in line with baseline prototype V5

### PR DESCRIPTION
**Pull request by AimeeHeywood**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -49,35 +49,35 @@ unauthorised.heading = You can’t access this service with this account
 eori.number = EORI number
 eori.details = EORI details
 
-registeredAddressForEori.title = This EORI is associated with the Government Gateway Account
-registeredAddressForEori.heading = This EORI is associated with the Government Gateway Account
-registeredAddressForEori.businessName = Business name
+registeredAddressForEori.title = What are the registered details asscoiated with this EORI?
```
Comment (gokyo) [inline conf/messages.en]: typo: `asscoiated `

File: conf/messages.en [hunk 2]
```diff
@@ -49,35 +49,35 @@ unauthorised.heading = You can’t access this service with this account
...
+registeredAddressForEori.heading = What are the registered details associated with this EORI?
```
Comment (gokyo) [inline conf/messages.en]: I would add the word `number` at the end

File: conf/messages.en [hunk 3]
```diff
@@ -49,35 +49,35 @@ unauthorised.heading = You can’t access this service with this account
...
-whichBestDescribesYou.option1 = I am an employee or owner of the business or organisation
-whichBestDescribesYou.option2 = I am a representative applying for a different business or organisation
-whichBestDescribesYou.checkYourAnswersLabel = Which of these best describes you?
-whichBestDescribesYou.error.required = Select which of these best describes you?
+whichBestDescribesYou.title = Who is this application for?
+whichBestDescribesYou.heading = Who is this application for?
+whichBestDescribesYou.option1 = The business or organisation that I own or work for
+whichBestDescribesYou.option2 = A different business or organisation that I am representing
+whichBestDescribesYou.checkYourAnswersLabel = Who is this application for?
+whichBestDescribesYou.error.required = Select who this application is for
```
Comment (gokyo) [inline conf/messages.en]: `?` at the end of the sentence

File: conf/messages.en [hunk 4]
```diff
@@ -90,43 +90,41 @@ registerBusinessRepresenting.error.field2.required = Enter field2
...
 registerBusinessRepresenting.error.field2.length = field2 must be 100 characters or less
 
-enterContactDetails.title = Enter contact details
-enterContactDetails.heading = Enter contact details
+enterContactDetails.title = Who is the contact for this application?
+enterContactDetails.heading = Who is the contact for this application?
 enterContactDetails.field1 = Full name
 enterContactDetails.field2 = Email address
-enterContactDetails.field3 = Direct telephone number
+enterContactDetails.field3 =  Telephone number (optional)
```
Comment (gokyo) [inline conf/messages.en]: remove extra space at the beginning

File: conf/messages.en (hunk 1 above)
Comment (ed0906) [reply conf/messages.en]: @AimeeHeywood You had one job!

File: conf/messages.en [hunk 5]
```diff
@@ -90,43 +90,41 @@ registerBusinessRepresenting.error.field2.required = Enter field2
...
-previousCommodityCode.paragraph.changes = It is important that you update this application with any changes to the goods.
-previousCommodityCode.details.summary = What is my commodity code ruling reference
-previousCommodityCode.details.contentParagraph = This reference number was supplied on the document when you were given your original commodity code ruling. This was known as a BTI ruling for applications before January 2019.
+previousCommodityCode.title = What is the reference number for the BTI ruling you want to renew?
+previousCommodityCode.heading = What is the reference number for the BTI ruling you want to renew?
+previousCommodityCode.field1 = BTI ruling reference number
+previousCommodityCode.checkYourAnswersLabel = BTI ruling reference number 
+previousCommodityCode.error.field1.required = Enter previous BTI ruling reference number
+previousCommodityCode.error.field1.length = Previous BTI ruling must be 100 characters or less
+previousCommodityCode.details.summary = What is my BTI ruling reference number
```
Comment (gokyo) [inline conf/messages.en]: `?` at the end

File: conf/messages.en [hunk 6]
```diff
@@ -90,43 +90,41 @@ registerBusinessRepresenting.error.field2.required = Enter field2
...
 informationAboutYourItem.checkYourAnswersLabel = Information about your item
-informationAboutYourItem.error.required = Select if you have information about your item
-informationAboutYourItem.listItem1 = Confidential information
-informationAboutYourItem.listItem2 = Describe your item
-informationAboutYourItem.listItem3 = Upload photographs, manuals and other supporting information
-informationAboutYourItem.listItem4 = Commodity code (optional)
-informationAboutYourItem.listItem5 = When to send a sample to the classification team
+informationAboutYourItem.error.required = Select yes or no
+informationAboutYourItem.listItem1 = any confidential information you do not want to be published on the BTI ruling website
+informationAboutYourItem.listItem2 = a detailed description of your item to help to determine its classfication
```
Comment (gokyo) [inline conf/messages.en]: remove extra space at the end of the sentence

File: conf/messages.en [hunk 7]
```diff
@@ -49,35 +49,35 @@ unauthorised.heading = You can’t access this service with this account
 eori.number = EORI number
 eori.details = EORI details
 
-registeredAddressForEori.title = This EORI is associated with the Government Gateway Account
-registeredAddressForEori.heading = This EORI is associated with the Government Gateway Account
```
[the remark this answers is not in the document]
Comment (AimeeHeywood) [reply conf/messages.en]: Hi Ed, not sure what you mean by this one?

[the remark this answers is not in the document]
Comment (gokyo) [reply conf/messages.en]: I think he commented in a wrong PR?!
@ed0906

File: conf/messages.en [hunk 8]
```diff
@@ -145,18 +143,18 @@ describeYourItem.listItem2 = how it works
 describeYourItem.listItem3 = what it is made from
 describeYourItem.listItem4 = how it is packaged
 
-confidentialInformation.title = Provide information you want to remain confidential
-confidentialInformation.heading = Provide information you want to remain confidential
-confidentialInformation.field1 = Confidential information
+confidentialInformation.title = What information do you want to keep confidential?
+confidentialInformation.heading = What information do you want to keep confidential?
+confidentialInformation.field1 = This information will not be published on the BTI ruling website
```
Comment (gokyo) [inline conf/messages.en]: `published on`? or `published to` ?

File: conf/messages.en [hunk 9]
```diff
@@ -90,49 +90,47 @@ registerBusinessRepresenting.error.field2.required = Enter field2
...
+previousCommodityCode.error.field1.length = Previous BTI ruling must be 100 characters or less
+previousCommodityCode.details.summary = What is my BTI ruling reference number?
+previousCommodityCode.details.contentParagraph = This reference number was supplied on the document when you were given your original BTI ruling.
 
 describeYourItem.title = Describe your item
 describeYourItem.heading = Describe your item
 describeYourItem.field1 = Item name
 describeYourItem.field2 = Describe your item in detail
-describeYourItem.field2.hint = Information in this box will be made public
+describeYourItem.field2.hint = This information may be published on the BTI website
```
Comment (gokyo) [inline conf/messages.en]: `published on`? or `published to` ?

File: conf/messages.en [hunk 10]
```diff
@@ -90,49 +90,47 @@ registerBusinessRepresenting.error.field2.required = Enter field2
...
-informationAboutYourItem.listItem2 = Describe your item
-informationAboutYourItem.listItem3 = Upload photographs, manuals and other supporting information
-informationAboutYourItem.listItem4 = Commodity code (optional)
-informationAboutYourItem.listItem5 = When to send a sample to the classification team
+informationAboutYourItem.error.required = Select yes or no
+informationAboutYourItem.listItem1 = any confidential information you do not want to be published on the BTI ruling website
+informationAboutYourItem.listItem2 = a detailed description of your item to help to determine its classfication
+informationAboutYourItem.listItem3 = photographs, manuals, technical specifications and other information to support your description and help to speed up the application
+informationAboutYourItem.listItem4 = a commodity code if you have found one
+informationAboutYourItem.listItem5 = a sample if it is needed
```
Comment (gokyo) [inline conf/messages.en]: how about `a sample if needed`? is it easier to read?

File: conf/messages.en (hunk 8 above)
Comment (AimeeHeywood) [reply conf/messages.en]: We'll stick with 'published on' I think as that's what users would say

File: conf/messages.en (hunk 10 above)
Comment (AimeeHeywood) [reply conf/messages.en]: Ok

Comment (AimeeHeywood) [reply conf/messages.en]: Always happy to lose words

**Pull request merged**